src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Predicates/PredicateQuery.cs

```diff
@@ -60,8 +65,21 @@ public string GetColumnName(string propertyPath)
                 {
                     // Return the default alias
                     // ContentItemId -> ContentItemIndex.ContentItemId
-                    _usedAliases.Add(alias);
-                    return Dialect.QuoteForTableName($"{_tablePrefix}{alias}") + "." + Dialect.QuoteForColumnName(values[0]);
+                    // Ensures its actually a property of ContentItemIndex and gets the correct casing.
+                    var propertyName = values[0].ToLower();
+
+                    var indexKey = nameof(ContentItemIndex) + propertyName;
```

Comment:
Didn't realize it was for a single type. Please use a `Dictionary<string, string>(StringComparison.OrdinalIgnoreCase)` then and pre-initialize it in a static contructor.

Reply:
Let me do it and you'll review it after

Reply:
@sebastienros sure, i mean this is what i was gonna do but @Jetski5822 had other ideas hahaa